tests/components/brother/test_sensor.py

```diff
@@ -208,9 +214,10 @@ async def test_sensors(hass):
 
     state = hass.states.get("sensor.hl_l2340dw_uptime")
     assert state
-    assert state.attributes.get(ATTR_ICON) == "mdi:timer-outline"
-    assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == TIME_DAYS
-    assert state.state == "48"
+    assert state.attributes.get(ATTR_ICON) is None
+    assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) is None
+    assert state.attributes.get(ATTR_DEVICE_CLASS) == DEVICE_CLASS_TIMESTAMP
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6072450)